doc/release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,75 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+## Supported platforms
+
+OpenJDK binaries that contain the Eclipse OpenJ9 virtual machine are available from the
+[AdoptOpenJDK community](https://adoptopenjdk.net/?variant=openjdk8-openj9). These builds are
+being tested against the OpenJDK test suite.
```

Comment:
@smlambert Can you provide some details on the levels of testing for OpenJ9?  I know there's a mix of FV, SV, and OpenJDK tests being run at one or both of the OpenJ9 and Adopt Jenkins.  A little more detail for the release notes would help.

Reply:
These builds are being tested against several test suites, including the OpenJDK regression tests, the AdoptOpenJDK and OpenJ9 System Verification (SV) tests, the OpenJ9 Functional Verification (FV) tests, and the Java Compatibility Kit (JCK) tests.

Reply:
Note: We are additionally running some 3rd party application testing (a set that will grow by our next release) and some perf benchmarks, but we are not gating releases (https://github.com/AdoptOpenJDK/openjdk-tests/issues/186) on these two categories of tests.